server/service/auth/manager.go

```diff
@@ -17,14 +17,15 @@ package auth
 
 import (
 	"fmt"
+	"github.com/apache/servicecomb-service-center/datasource"
 	"github.com/apache/servicecomb-service-center/pkg/log"
 )
 
-type dataSourceEngine func(opts Options) (DataSource, error)
+type dataSourceEngine func(opts Options) (datasource.DataSource, error)
```

Comment:
I think that plugin mechanism is damaged if plugin's implemention was migrated to service/datasource pkg but leaving plugin manager where it stays before. This will lead to module coupled in my opinion. Refer to https://github.com/go-chassis/go-chassis/tree/master/pkg/backends/quota.

Reply:
Just leave the ms,dep,auth decoupled as three fully seperated plugins or aggreate them to one plugin.

Reply:
Yes, i am going to mv plugin mgr to datasource pkg. I will finish it in Next PR

Reply:
ok~~LGTM